docs/README.md

```diff
@@ -41,17 +41,12 @@ Review the [Change Log](ChangeLog.md) page for a history of changes in each rele
 
 The Splunk Operator should work with any [CNCF certified distribution](https://www.cncf.io/certification/software-conformance/) of Kubernetes. We do not have platform recommendations, but this is a table of platforms that our developers, customers, and partners have used successfully with the Splunk Operator.
 
-|  |   |
-| ---- | --- |
-| Splunk Development & Testing Platforms | Amazon Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE) |
-|Customer Reported Platforms | Microsoft Azure Kubernetes Service (AKS), Red Hat OpenShift |
- Partner Tested Platforms | HPE Ezmeral|
-| Other Platforms |[CNCF certified distribution](https://www.cncf.io/certification/software-conformance/) |
-|  |   |
-
-
-
-
+<table>
+<tr><td> Splunk Development & Testing Platforms </td><td> Amazon Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE) </td></tr>
+<tr><td> Customer Reported Platforms </td><td> Microsoft Azure Kubernetes Service (AKS), Red Hat OpenShift </td></tr>
+<tr><td> Partner Tested Platforms</td><td> HPE Ezmeral</td></tr>
+<tr><td> Other Platforms </td><td>CNCF certified distribution</td></tr>
```

Comment:
The hyperlink needs to be added back?

Reply:
I removed it as we already have CNCF with link in the section above.